.github/workflows/release-please.yml

```diff
@@ -1,79 +1,55 @@
+permissions:
+  contents: write
+  pull-requests: write
 jobs:
   release-please:
+    name: Release Please
     runs-on: ubuntu-latest
-    outputs:
-      release_created: ${{ steps.release.outputs.release_created }}
-      sha: ${{ steps.release.outputs.sha }}
-      upload_url: ${{ steps.release.outputs.upload_url }}
-    # https://github.com/google-github-actions/release-please-action in case more config is needed
     steps:
       - uses: google-github-actions/release-please-action@v3
         id: release
         with:
+          release-type: node
           command: manifest
-          # In case we need to add a PAT and have the pr's trigger actions
-          #token: ${{ secrets.PAT }}
-          # See how to configure file https://github.com/googleapis/release-please/blob/main/docs/manifest-releaser.md
-          # See output info at https://github.com/google-github-actions/release-please-action#configuration for variables
-          # Trigger more steps using "if: ${{ steps.release.outputs.release_created }}"
-
+    outputs:
+      tag: ${{ steps.release.outputs.tag_name }}
+      release_created: ${{ steps.release.outputs.release_created }}
+      sha: ${{ steps.release.outputs.sha }}
+      upload_url: ${{ steps.release.outputs.upload_url }}
+    # Need to allow GitHub Actions to create and approve pull requests.
   build:
-    needs: release-please
     name: "Build"
-
-    runs-on: ${{ matrix.os }}
-
-    if: ${{ needs.release-please.outputs.release_created }}
-
-    strategy:
-      matrix:
-        include:
-          - os: ubuntu-latest
-            name: linux
-            yarn-task: build:linux
-          - os: macos-latest
-            name: mac
-            yarn-task: build:mac
-          - os: windows-latest
-            name: win
-            yarn-task: build:win
-
+    needs: release-please
+    if: needs.release-please.outputs.release_created
+    uses: ./.github/workflows/build.yml
+  publish:
+    name: Publish
+    needs:
+      - build
+      - release-please
+    runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - name: Use Node.js 16.x
-        uses: actions/setup-node@v2
-        with:
-          node-version: 16.x
-          cache: "yarn"
-      - name: Install NPM Packages
-        run: |
-          yarn install --immutable --network-timeout 120000
-      - name: Build
-        run: yarn ${{ matrix.yarn-task }}
-      # Possibly set up some basic unit testing just to make sure parts render and none of the libraries are straight up breaking
-      - uses: actions/upload-artifact@v2
-        with:
-          name: built-app-${{ matrix.name }}
-          path: |
-            app/main/dist/*.*
-      - name: Upload Release Asset(s)
+      - name: Download artifact
+        uses: actions/download-artifact@v3
+      - name: Upload Release Assets
         id: upload-release-asset
-        uses: sekwah41/upload-release-assets@v1.1.0
+        uses: softprops/action-gh-release@v0.1.15
```

Comment:
Ah good, I threw together one to fix an issue and hadn't come across this action.

Reply:
Thanks!